examples/substrate-to-evm-react-example/README.md

```diff
@@ -0,0 +1,80 @@
+# Substrate to EVM example
```

Comment:
I do not see any example here. Where is the code?

Reply:
This is the README to run the example. The example itself is the whole folder is an example of a frontend application.